rotkehlchen/db/dbhandler.py

```diff
@@ -2104,7 +2110,7 @@ def delete_data_for_ethereum_address(self, write_cursor: 'DBCursor', address: Ch
         loopring.remove_accountid_mapping(write_cursor, address)
 
         dbtx = DBEvmTx(self)
-        dbtx.delete_transactions(write_cursor=write_cursor, address=address, chain=SupportedBlockchain.ETHEREUM)  # noqa: E501
+        dbtx.delete_transactions(write_cursor=write_cursor, address=address, chain=blockchain.to_chain_id())  # type: ignore[arg-type]  # noqa: E501
```

Comment:
Wtf are you doing here? Why are you:
1. Converting blockchain to chain_id while the function accepts a `SupportedBlockchain`?
2. Mypy sees you are doing this and warns you and then you suppress the error?!

```suggestion
        dbtx.delete_transactions(write_cursor=write_cursor, address=address, chain=blockchain.to_chain_id())  # type: ignore[arg-type]  # noqa: E501
```

Reply:
Yes, was a mistake :), fixed